src/cli_plugin/install/pack.js

```diff
@@ -17,7 +17,7 @@ async function listPackages(settings) {
     .map(file => file.replace(/\\/g, '/'))
     .map(file => file.match(regExp))
     .compact()
-    .map(([ file, _, folder ]) => ({ file, folder }))
```

Comment:
Another thing that eslint complains about. If we really don't like this style I think we should just stop destructuring the arg.